src/styles/globals.css

```
@@ -808,3 +808,65 @@ input::placeholder {
+
+/* ─────────────────────────────────────────────────────────────────
+   Share-asset stamp — real postage-stamp appearance for the
+   ShareAssetD3 component. Scalloped perforated edges, inner thin
+   frame, paper-warm body, hard shadow. Override `--stamp-bg` per
+   parent so the perforation dots match the canvas behind it.
+
+   Used in: src/components/Card/share-asset/ShareAssetD3.tsx
+   ───────────────────────────────────────────────────────────────── */
+.stamp {
+    --stamp-bg: #ffc900;
+    position: relative;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)